src/resmom/linux/alps.c

```diff
@@ -4447,6 +4605,7 @@ alps_request_parent(int fdin)
 		free_basil_response_data(brp);
 		return (NULL);
 	}
+
```

Comment:
I don't like blank lines. Didn't like them more than a decade ago, and I still don't like them.

Reply:
I'll remove it